fern/docs/pages/recipes/list-llm.mdx

```diff
@@ -81,4 +81,15 @@ Context: None
 Situation: tu es au milieu d'un champ.
 Tache: va a la rivière, en bas du champ.
 Décrit comment aller a la rivière.
+```
+
+### Optimised Models
+GodziLLa2-70B LLM (English, rank 2 on HuggingFace OpenLLM Leaderboard), bge large Embedding Model (rank 1 on HuggingFace MTEB Leaderboard)
```

Comment:
These ranking are valid at a given point in time (today), and it might change somewhere in the future.
Could you either reference that time point of reference here, or maybe be more evasive regarding its ranking (and maybe adding a link to these rankings)?

Reply:
Also is there a command to preview the fern docs in Local?
Wanted to make sure it looks good...

Reply:
Previews not working at the moment because you'd need a FERN API key that is not being passed to fork PRs. Looking into options.

Reply:
It does seems better to me yes 👍

Reply:
Looks better indeed